recipes/stactools/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "stactools" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stactools-{{ version }}.tar.gz
+  sha256: 691dc0038d3e709462748747d2e39c2f369cf16d5cd890faba0025e4424b4991
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+  - stac = stactools.cli:cmd
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - aiohttp >=3.8
+    - click >=8.1.3
+    - fsspec >=2021.7
+    - lxml >=4.6
+    - numpy >=1.21.0
+    - pyproj >=3.0
+    - pystac >=1.2
+    - python >=3.7
+    - rasterio >=1.2.9
+    - requests >=2.20
+    - shapely >=1.6
+    - stac-check >=1.2.0
+    - stac-validator >=3.1.0
+
+test:
+  imports:
+    - stactools
+  commands:
+    - pip check
```

Comment:
Could you please add `stac --help` to the tests?

Reply:
Added